### PR TITLE
PE-763: Dependency management for bundles

### PR DESCRIPTION
**Pull request by fedellen**
This PR adds `arbundles`, bumps `arweave` to be compatible with `arbundles`, and removes `arweave-bundles` to prepare `ardrive-core-js` for bundle compatibility

### CODE REVIEW

**Review by jdaev — APPROVED** [review]
Looks good 👍🏽

**Review by arielmelendez — APPROVED** [review]

File: src/community/smartweave_contract_oracle.ts [hunk 1]
```diff
@@ -12,6 +12,8 @@ export class SmartweaveContractReader implements ContractReader {
 
 	/** Fetches smartweave contracts from Arweave with smartweave-js */
 	async readContract(txId: TransactionID): Promise<unknown> {
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
```
Comment (matibat) [inline src/community/smartweave_contract_oracle.ts:16]: What are we ignoring here?

**Review by matibat — APPROVED** [review]
It looks good!
Left one single Q

File: src/community/smartweave_contract_oracle.ts (hunk 1 above)
Comment (fedellen) [reply src/community/smartweave_contract_oracle.ts:16]: Its silencing a type error. Smartweave is expecting an arweave `1.10.16`, but we now have version `1.10.18`.  Updating smartweave does not seem to resolve the error and the package is still working.

**Pull request merged**